pkg/base1/cockpit.js

```diff
@@ -2635,21 +2635,29 @@ function full_scope(cockpit, $, po) {
 
         var user = cockpit.user;
         var group = null;
+        var superuser = false;
 
         if (options)
             group = options.group;
 
+        if (options && options.superuser)
+            superuser = true;
```

Comment:
I think this option should be called 'admin'. Admin users typically can become a superuser.